snuba/web/rpc/v1/resolvers/R_eap_spans/common/aggregation.py

```diff
@@ -37,6 +37,8 @@
 
 CUSTOM_COLUMN_PREFIX = "__snuba_custom_column__"
 
+_FLOATING_POINT_PRECISION = 8
```

Comment:
A `Float32` would provide us with at least 8 digits of precision already I think, we should extend that to 12 if we're going to support `Float64`.